### DISCUSSION TITLE
START HERE

### DISCUSSION THREAD

**Discussion by vincenzospaghetti** [helium/HIP#667] [Announcements]
## What is the HIP Discussion Forum?
Welcome to the HIP Discussion Forum! This platform serves as a dedicated space for sharing ideas, thoughts, and processes aimed at enhancing the Helium Network and Helium Network Governance.

Previously, discussions took place on Discord, which made it challenging to keep conversations organized and have deliberative arguments. The HIP Discussion Forum addresses this issue by providing a structured environment for thoughtful discussion, and debate on HIPs (Helium Improvement Proposals), and critical network matters while still maintaining transparent record-keeping through GitHub.

The main goal of this forum is to create a collaborative space where ideas can be logged and improved upon over time. It enables community members to brainstorm enhancements, gather support, and allocate resources as they become available. Even those new to Helium can easily navigate and learn from these forums.

## Using the HIP Discussion Forum
First and foremost, the HIP Discussion Forum is an inclusive space for everyone. It is the place where ideas receive recognition, and all participants will encourage participation from anyone, regardless of their level of knowledge or the complexity of their ideas.

**The only guideline is that discussions on this forum should be HIP-relevant, meaning they should pertain to ideas for changing aspects of how Helium operates. For general Q&A, onboarding resources, and casual conversations, it is recommended to join the [Helium Discord](https://www.discord.com/helium/gg).**

To get started, community members should consider the scope of their idea and post it in the appropriate channel:

1. Announcements: Updates from maintainers
2. Helium Improvement Proposals: This is the main Forum to discuss active HIPs. HIPs will be added here for discussion when they have moved from the Draft phase to the Disucssion phase.
3. Enhancements - Communications: Share and store ideas for enhancing communications, announcements, and best practices within the Helium governance ecosystem.
4. Enhancements - Process: Share and store ideas for enhancing the quality, transparency, and accessibility of Helium governance processes.
5. Enhancements - Voting: Share and store ideas for enhancing the quality, transparency, and accessibility of Voting in the Helium Community.
6. Enhancements - Technical: Discuss ideas for technical protocol enhancements, which are the most common type of HIP.
7. Q+A: If you have questions about Helium governance or the HIP process, this is the perfect channel to ask.
8. Executive Summaries - HIPs: A channel for executive summaries of HIPs before they go up for voting.
9. Governance Managers & HIP Editors: A channel for those actively involved in governance to collaborate, deliberate, or come to consensus on changes to governance.

Please note that the Forum is an optional tool for sharing and storing ideas until they are prepared to become HIP proposals. You are welcome to create your own branch of the repository and initiate a HIP Pull Release when you and your team are ready to write a HIP draft and start the HIP process.

The HIP Issues Log is where all active HIPs reside. No idea is considered for acceptance until a HIP Draft is authored and pushed to the Issues log.

**Comment by A1iAshoor** [comment]
Lets do this. Voted.

**Comment by JKJ1000** [comment]
Holo veo que te gusta este Toke HNT

**Comment by clavisound** [comment]
Discord is closed. Reddit is the alternative?
